nstar/Cargo.toml

```diff
@@ -15,6 +15,7 @@ itertools = "0.9.0"
 log = "0.4.11"
 north = { path = "../north", features = ["api"], default-features = false }
 prettytable-rs = "0.8.0"
+regex = "1.4.1"
```

Comment:
other packages are using "1.3.9", should we use the same here?

Reply:
Technically this is not a problem but the double compile (in case building the rt for the host) is not nice. I'll update the other.